app/app.go

```diff
@@ -4,6 +4,7 @@
 package app
 
 import (
+	"crypto/ecdsa"
```

Comment:
Why ecdsa?

Reply:
Only other practical option is RSA. And the only compelling factor in the decision was signature length. RSA signatures are much longer for comparable security. So ECDSA gives us prettier URLs. :P

Reply:
😆 OK! Pretty URLs it is!